### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by momintlh**
Based on #33 
Updated the signature and fixed the callbacks. Did 2 build tests.

### CODE REVIEW

File: Assets/Plugins/PlayroomPlugin.jslib [hunk 1]
```diff
@@ -23,6 +23,16 @@ mergeInto(LibraryManager.library, {
       });
     }
 
+    function OnLaunchCallBack() {
```
Comment (SaadBazaz) [inline Assets/Plugins/PlayroomPlugin.jslib:26]: Did you test this @momintlh ?

File: Assets/Plugins/PlayroomPlugin.jslib [hunk 2]
```diff
@@ -23,6 +23,16 @@ mergeInto(LibraryManager.library, {
...
+      dynCall("v", onLaunchCallBack, []);
+    }
+
+    function OnDisconnectCallback() {
```
Comment (SaadBazaz) [inline Assets/Plugins/PlayroomPlugin.jslib:30]: +1 Did you test this @momintlh ?

File: Assets/Plugins/PlayroomPlugin.jslib (hunk 1 above)
Comment (momintlh) [reply Assets/Plugins/PlayroomPlugin.jslib:26]: Yes, I did, its working fine @SaadBazaz.

Comment (SaadBazaz) [reply Assets/Plugins/PlayroomPlugin.jslib:26]: Looks good! Resolving.

**Pull request merged**